### DISCUSSION TITLE
Different yt-dlp versions

### DISCUSSION THREAD

**Discussion by cashewFanfare** [Tzahi12345/YoutubeDL-Material#775] [Ideas]
Hi,

First, thanks for the project, seems like it's the best implementation of a web app for yt-dlp.  I've been having an issue with the yt-dlp release version being too old to access some of the supported sites.  Is there a way to choose to use the master branch of yt-dlp rather than the release version?

**Comment by Tzahi12345** [comment]
No because we don't build yt-dlp ourselves. We just download the release version. They do have a `nightly` but they don't give an explicit link to it, I'm sure it's somewhere: https://github.com/yt-dlp/yt-dlp#update
